src/lib.rs

```diff
@@ -10,6 +10,7 @@ extern crate futures_util;
 #[cfg(test)]
 extern crate futures_executor;
 
+pub mod broadcaster;
```

Comment:
It should export the `Broadcaster` from inside of the `signal` module, i.e. users should import it as `futures_signals::signal::Broadcaster`.

Reply:
Hmm ok. Are you happy with the file structure as-is, or should I e.g. rearrange as `signal/mod.rs`, `signal/signal.rs`, `signal/broadcaster.rs`? I was wary of adding to the existing `signal.rs` file as I didn't want it to have too many separate ideas in it.